membership-attribute-service/app/services/PaymentFailureAlerter.scala

```diff
@@ -89,15 +89,16 @@ object PaymentFailureAlerter extends LoggingWithLogstashFields with StrictLoggin
       expectedAlertText.map { someText => shouldShowAlert.option(someText).flatten }
     }
   }
-  val alertableProducts = List(Product.Membership, Product.Contribution, Product.Digipack)
+
+  val nonAlertableProducts: List[Product] = List()
```

Comment:
Oh cool! So there are no non-alertable products now? That's great.

Reply:
Exactly. As far as I'm aware there aren't currently any products which we want to prevent alerting for.